src/pymor/analyticalproblems/expressions.py

```diff
@@ -324,15 +398,29 @@ class UnaryFunctionCall(Expression):
 
     numpy_symbol = None
 
-    def __init__(self, *arg):
-        if len(arg) != 1:
-            raise ValueError(f'{self.numpy_symbol} takes a single argument (given: {arg})')
-        self.arg = _convert_to_expression(arg[0])
+    _parameters_varargs_warning = False  # silence warning due to use of *args in __init__
+
+    def __init__(self, arg, *args):
+        if args:
+            raise ValueError(f'{self.numpy_symbol} takes a single argument (given: {(arg,) + args})')
+        self.arg = _convert_to_expression(arg)
         self.shape = self.arg.shape
 
     def numpy_expr(self):
         return f'{self.numpy_symbol}({self.arg.numpy_expr()})'
 
+    def fenics_expr(self, params):
+        import ufl
+        if self.fenics_op is not None:
```

Comment:
It's better to write

```python
if self.fenics_op is None:
    raise NotImplementedError(f'UFL does not support operand {self.numpy_symbol}')
...
```
Then you can continue without having to worry about the `else`-clause and without any additional indentation.